### ISSUE TITLE
Add lodestar validator client

### ISSUE THREAD

**Issue by corverroos** [ObolNetwork/charon-distributed-validator-cluster#23]
## Problem to solve

Charon states it supports Lodestar validator client. But this hasn't been tested or confirmed yet. This repo also only had teku and lighthouse.

## Proposed solution:

- Add lodestar as VC for node2. 
- First prize is implemeting it the same as teku (without a folder or run script), just a docker-compose service.
- Remember to configure network as auto, since charon can connect to prater/gnosis/kiln/simnet/devnet.
- Remember to load the keys on startup.
- If this is not trival, like lighthouse, a folder and run script can be added.
- Try to keep everything to the absolute minimum

**Comment by dB2510** [comment]
Blocked by https://github.com/ObolNetwork/charon/issues/473

**Comment by OisinKyne** [comment]
Custom preset requirements for simnet are now out of scope for this template repo. We can move forward with lodestar support without it (and without custom dockerfiles/run scripts 🤞 ) . Pulling out of parking lot.

**Comment by dB2510** [comment]
Current status of this issue:
- Lodestar uses an optimization to use a single attestation data for all committees is a probable blocker ito supporting it.
-  - The beacon spec provides NO guarantees that attestation datas for different committees MUST be identical. 
 - In the event that charon stores different attestation datas in the dutydb (for the same slot), then lodestar will basically attest with a SINGLE attestation data while other VCs will attest using the different datas, resulting in aggregation failure. 
 - It might be that that in 99% of the time, all attestation datas are actually the same for different committees, it is however only an accidental implementation phenomenon which might change at anypoint due to changes in implementation or race conditions in the code.
 - We are going to monitor whether we can detect inconsistent attestation data for different committees, which will prove that we cannot support lodestar.
 - If we cannot detect inconsistencies, then we are in a tough spot, since we can provisionally support lodestar with the caveat that attestation might fail at any point.

**Comment by dB2510** [comment]
To track with lodestar repo: https://github.com/ChainSafe/lodestar/issues/4687

**Comment by OisinKyne** [comment]
Can we close this when lodestar v1.10 comes out? Or can we close it already even?
